save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/demo/diktat/DiktatDemoTool.kt

```diff
@@ -2,16 +2,19 @@ package com.saveourtool.save.demo.diktat
 
 /**
  * Enum that represents possible tool names in diktat-demo
+ * @property owner
+ * @property toolName
+ * @property version
  */
-enum class DiktatDemoTool {
+enum class DiktatDemoTool(val owner: String, val toolName: String, val version: String) {
     /**
      * Run diktat
      */
-    DIKTAT,
+    DIKTAT("saveourtool", "diktat", "v1.2.3"),
```

Comment:
Could we move versions in some common place? You use it in 3 places

Reply:
It is planned to use database for that (create a new scheme like it is done in `save-sandbox`)